addOns/graphql/src/main/java/org/zaproxy/addon/graphql/ExtensionGraphQl.java

```diff
@@ -48,6 +48,7 @@ public class ExtensionGraphQl extends ExtensionAdaptor
         implements CommandLineListener, SessionChangedListener {
 
     public static final String NAME = "ExtensionGraphQl";
+    static final int PLUGIN_ID = 50007;
```

Comment:
I'd call it `SCAN_RULE_ID`.

Reply:
That's a good point, "PLUGIN" was an old form. I can do a PR tweaking this across the board sometime.

Reply:
But this ID is not for a single scan rule, but for all non-scan-rule alerts raised by this add-on. Perhaps it could be called `TOOL_ALERT_ID`?

Reply:
I’m good with that in this case.